.teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt

```diff
@@ -23,5 +23,8 @@ open class BaseGradleBuildType(model: CIBuildModel, val stage: Stage? = null, us
                 artifacts(days = 7)
             }
         }
+        params {
+            param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%")
```

Comment:
Do we need to pass that key now for every build? Why don't we set it on the parent project?

Reply:
We should not set such an access key at all. But instead use the API that allows to publish automatically when authenticated - just like Spring does it. This will also make it very concenient for building locally, either as a Gradle developer with an access (scans will be published) or an external developer (no scans will be published). @facewindu can give more insights if needed.

Reply:
Actually, I just realized this is on the TC config. There, it might make sense to set it on the top-most parent project. The rest I wrote about automatic publishing only when authenticated remains.

Reply:
I put it in `RootProject`.